### ISSUE TITLE
{Linked:Bug2611817}[Accessibility] Talkback focus moves two times on the element which has select action

### ISSUE THREAD

**Issue by longlong-cs** [microsoft/AdaptiveCards#7864]
### Target Platforms

Android

### SDK Version

2.8.2

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: Samsung Galaxy S20
Android Version: 12
App name: Teams Production Build
App version: 1416/1.0.0.2022363703/0824

**Repro Steps:**

- Launch MS teams and login with valid credentials.
- Activate talkback.
- Navigate to 'Cortana Voice Activation' button and activate it.
- Cortini popup dialog will be open.
- Try to Give Command "Join my next meeting".
- Navigate to the 'Change meeting settings' button using swipe gesture.
- Observe talkback focuses on lands on two times on the control or not.

**Actual Results:** 
After talkback focus lands on the 'Change meeting settings' button and after next swipe focus moves to "change meeting settings" again but this time talkback does not announce the button role information.

**Expected Result:**
Talkback focus should not move two times on the "Change meeting settings" button present in "Ready to join meeting" page. As after using swipe one time focus moves to 'Change meeting settings' button only and after another swipe focus moves to the next control.

**Note: this issues can be reproduced on AC sample app with a very simple json payload**

MAS Reference: 
[MAS 1.3.2 – Meaningful Sequence](https://aka.ms/MAS1.3.2)

### Screenshots

First focus, announce "Change Meeting Settings, button"
![image](https://user-images.githubusercontent.com/66937113/190326622-64b3c722-061d-47e6-9cd4-314563fe0bcf.png)

Second focus, announce "Change Meeting Settings"
![image](https://user-images.githubusercontent.com/66937113/190326661-4d5e4cae-1231-487d-b49a-4defca0f42f0.png)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "version": "1.0",
    "speak": "Test Meeting title",
    "body": [
        {
            "type": "Container",
            "items": [
                {
                    "type": "ColumnSet",
                    "columns": [
                        {
                            "type": "Column",
                            "width": "stretch",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "size": "large",
                                    "text": "Test Meeting title",
                                    "wrap": true,
                                    "maxLines": 2,
                                    "spacing": "medium"
                                },
                                {
                                    "type": "TextBlock",
                                    "isSubtle": true,
                                    "text": "Tomorrow,  12:05 AM - 1:00 AM",
                                    "spacing": "none"
                                }
                            ]
                        }
                    ]
                }
            ]
        },
        {
            "type": "Container",
            "items": [
                {
                    "type": "TextBlock",
                    "size": "medium",
                    "isSubtle": true,
                    "text": "Meeting settings",
                    "spacing": "none"
                },
                {
                    "type": "ColumnSet",
                    "columns": [
                        {
                            "type": "Column",
                            "width": "auto",
                            "items": [
                                {
                                    "type": "Image",
                                    "url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAACxLAAAsSwGlPZapAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAJrSURBVHgB1ZnhcdowGIZfqVyvP+kGjJBOAF2jTYI7QicoTFA6QdWE3PVfmglCNmAEdwP/aH7FWP0+J5dziGwkbEvykwsHssQ9r/3ZRpbAwEiSZIIiv6a3E2i9FBgQj/IPt4CYPDVlEgPBIE/obBBHoEY+hdx9jD5Ak7xSv9OoAxyS50/i9aDTGXZ6DiFn1HmCbtmiKO4wertSSqVNHW3kGVEZMMaOLk8CM/hhoS7WS9MGW3lG1A/oHwGd/Ly4+lVts5FPzk+/0ZF8z0dyVG7PHxaQ1QE+0Fst72+qLYfkn6sEVCWS7gBFPhVlzRe4hVf0FvKepP5ksJc/oe3X+1UikesEXnGX/3L+eU5721jikv6m8MYRe35+9l1DKGocm75R0Amh4QU3eeBdZnNVHMELzvLjcpuobjPj4cecm7wo3kzr6t1EzwHca76p3k30GMBWHnxH2+zfYW3pKYCDfEt6COBPnuk4gF95psMA/uWZjgKEkWc6CBBOnmkZIKw80yJAeHnmyABxyDNHBIhHnnEMEJc84xAgPnnGMkCc8oxFgHjlmQMBAshr/ZVeM9vuDQHC7Hl1ebWCzD+UkxwLagKELZtycsNzY43Nob4UQKQvm0KUjfi738Qh1OWaJvhYNg3l53N3lW8Ks+eL+j1ND4AXTeeFRFFOohGubHSq1utNU4+m80I+Df4Rqubp8fjCpp/5vNDmFRqPJ+yyLBFHkrNPCSU/gSxWhhUaL/IZr/GWpdGSFwF6lufyTOn/BvLfqlqubaguMVkv68REwxJT/PKMGLI8I4csz/BPicqT4GHJM5IvZ3i8QmyHJs/8B3OjcjOBk8MRAAAAAElFTkSuQmCC",
                                    "horizontalAlignment": "center",
                                    "altText": "camera",
                                    "width": "12px",
                                    "height": "12px"
                                }
                            ],
                            "verticalContentAlignment": "center"
                        },
                        {
                            "type": "Column",
                            "width": "auto",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "size": "small",
                                    "isSubtle": true,
                                    "text": "Video is off",
                                    "spacing": "none"
                                }
                            ],
                            "verticalContentAlignment": "center",
                            "spacing": "small"
                        },
                        {
                            "type": "Column",
                            "width": "auto",
                            "items": [
                                {
                                    "type": "Image",
                                    "url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAACxLAAAsSwGlPZapAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAPDSURBVHgB7Zm9UttAFIXPSoY25gnivEBwmS72EwS6DEMS0SUVblMBTxDcpUOAJ0MX6NIBT4CcpEiH3gBnJswwWNbmruQfCWullbRykpl8DWht756jvXv3j0ETlmXV4Xvb4GiBoUlF9fFHDnz/ErXlfdu2XWiGQQPW680d+tPBTHQynNswl/Z0GillgN56A/7wM1XTVP8Vd2EstXWZMFCG3OIFTJg+D8xroLCBMGzyip/AGhh5B9BAoRAKQ8e7Rll8tO1e7yKxjVcbFiWEZtbgL9YDo+EadMCwk1Qc9C4zDmAY2/SiroIMJ6FgCLEXqIhxRtuNFJH4+4bs+zUUIczzJeEOzNv1aEmCeEHXtj85slqKDuI6SkHijVtKpaeDSYlE/KF91Ouk1VQujRYil3grq7YFG9ArXrBAA/rFCxZkoBrxggUYqE68oGID1YoXVGigevGCigwoixfrIRslqMBADvEa0GxgseIFGg0sXrxAupgLl7AjWjabF9nbP33i87Wb1gP+8Ao+PwjW4xsbKatPzW9eud2QRAOWtdkKtn0hdZimpCK94tXbnVFiDGgSX/MHKIHEgOfGn/34+p/jVN+bX3ZRArUdGWON6KN93FPZSSnAXRqosx7wKGyir5TTGVIGiT1g2ycu/ZlVzLEqq6DUgGW4QFxNK/asEF7yMcDhRBpqJp0MlM7zbLT3oOR5RICbtheekDKI+Vm0smiHCDRMUnvjng7rC0/qZlnnYe9IkBswl+zwn0pmWLEC3Y2VeMPYM+PmKRQwZR84jnPXXH26QuK3NIvvkvi30YJx/v8wK6HwOeq9U6nMTPvQ6X/94jg/7qYNSdbzMLx1cLaSelbKKSQ4tiiDfURMvDimHIlD4ukYo/POjtP/1ocCymejKpsRy3rZgFdrUmA2KQU+pjj4SVF6jXvvzD6ZxXtc/PA8MvsGRslkG4ooGZCIvyTxLRQkUXxwdzBqRwd3FpkGUmO+wI3L9Cpq/kZnAMNvq6TOKKkGlAdsYIQd0szjxGZWRETDa8Lz1+jE+Q3mjyYLiRewnOIPKUYf0a/WIK/SnS4BmBAqBidvyL+fP2xirSUVZg3Y4PKBsZ14/OZG9FQXxq/9aJrOy5yBPEcfBY1oET4hycAN4jGaeW4T7JxqBq1j/CeUNsV6JjIfUIiIOYAxui9GX3alVJQkA1cRAbkPnYJZ1cf5tIAzmryObVTE/H5AzKq+0RETEInv4i9nzsA4G3Twj/AHbmj08t/APPEDAcbYDSrEhGYc5/sg2EcAz8KV5fF7VMhvu5soIvGe5TsAAAAASUVORK5CYII=",
                                    "horizontalAlignment": "center",
                                    "width": "12px",
                                    "height": "12px"
                                }
                            ],
                            "verticalContentAlignment": "center",
                            "spacing": "large"
                        },
                        {
                            "type": "Column",
                            "width": "auto",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "size": "small",
                                    "isSubtle": true,
                                    "text": "Mic is off",
                                    "spacing": "none"
                                }
                            ],
                            "verticalContentAlignment": "center",
                            "spacing": "small"
                        }
                    ],
                    "spacing": "none"
                },
                {
                    "type": "Container",
                    "items": [
                        {
                            "type": "TextBlock",
                            "size": "small",
                            "color": "accent",
                            "text": "Change meeting settings"
                        }
                    ],
                    "selectAction": {
                        "type": "Action.Submit",
                        "data": {
                            "intent:confirm": true,
                            "change_settings": "true"
                        },
                        "title": "Change meeting settings"
                    }
                }
            ],
            "spacing": "extraLarge"
        },
        {
            "type": "Container",
            "items": [],
            "spacing": "medium"
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "data": {
                "intent:confirm": true
            },
            "title": "Join"
        },
        {
            "type": "Action.Submit",
            "data": {
                "intent:cancel": true
            },
            "title": "Cancel"
        }
    ]
}
```

**Comment by JimXu99** [comment]
@paulcam206 , do we have an ETA? this will soon downgrade our feature's accessibility grading.

**Comment by anna-dingler** [comment]
Update:

This issue seems related to #7959 and PR #7554. Fixing #7959 will likely resolve this selectAction issue as well.

**Comment by longlong-cs** [comment]
@anna-dingler I see that #7554 has been fixed, so could you take a look at #7959 and help verify whether this can help resolve this issue? thanks!
